### ISSUE TITLE
R2018b empty sparse matrices do not match the format

### ISSUE THREAD

**Issue by ennerf** [HebiRobotics/MFL#35]
![image](https://user-images.githubusercontent.com/5491587/50539702-de057900-0b84-11e9-88aa-1a1a4f1dd931.png)

The [MAT-File Format](https://www.mathworks.com/help/pdf_doc/matlab/matfile_format.pdf) for 2018b specifies `Row Index (ir)` field to have `nzmax` values. For empty sparse matrices as well as matrices with a single element the `nzmax` value is always `1`.

However, in `R2018b` there seems to have been an undocumented behavior change in that the `Row Index` field is now empty, even though `nzmax` is still `1`.

This breaks the current parser.

**R2017b**
`EmptySparse = sparse(10,10);` 

```
[0] Compressed (61 bytes, position = 128)
  [0] Matrix (128 bytes, position = 0)
    [0] UInt32[2] = [4101, 1] // Sparse
    [1] Int32[2] = [10, 10]
    [2] Int8[11] = ['EmptySparse']
    [3] Int32[1] = [0]
    [4] Int32[11] = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
    [5] Double[0] = []
```

**R2018b**
`EmptySparse = sparse(10,10);` 

```
[0] Compressed (58 bytes, position = 128)
  [0] Matrix (128 bytes, position = 0)
    [0] UInt32[2] = [4101, 1] // Sparse
    [1] Int32[2] = [10, 10]
    [2] Int8[11] = ['EmptySparse']
    [3] Int32[0] = []
    [4] Int32[11] = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
    [5] Double[0] = []
```

**Comment by ennerf** [comment]
fixed in #36

**Comment by ennerf** [comment]
I've filed a bug report regarding the change in behavior and documentation mismatch.
